src/dataMap/metaManager/metaSchema.js

```diff
@@ -318,10 +324,13 @@ export default () => {
      * If the {@link ManualRowResize} or {@link AutoRowSize} plugins are enabled, this is also the minimum height that can
      * be set via either of those two plugins.
      *
+     * The default height for rows in the rendering process equals 23px.
      * Height should be equal or greater than 23px. Table is rendered incorrectly if height is less than 23px.
      *
+     * An `undefined` value is for detection in {@link Hooks#modifyRowHeight} hook if plugin or setting changed the default size.
+     *
      * @memberof Options#
-     * @type {number|number[]|string|string[]|Function}
+     * @type {number|number[]|string|string[]|undefined[]|Function}
```

Comment:
The same problem as above.

```suggestion
     * @type {number|number[]|string|string[]|Array<undefined>|Function}
```